frontend/src/views/WheresMyGene/components/Beta/index.tsx

```diff
@@ -1,18 +1,31 @@
-import { StyledCallout, SubmitIssue } from "./style";
+import { StyledNotification, SubmitIssue } from "./style";
 
 export default function Beta(): JSX.Element {
+  const position = {
+    position: "absolute",
+    bottom: "60px",
+    right: "15px",
+    "z-index": "1",
+  } as React.CSSProperties;
+
   return (
-    <StyledCallout intent="info" dismissed={false}>
-      This feature is in beta. We would appreciate your feedback, please fill
-      out a{" "}
+    <StyledNotification
+      intent="info"
+      autoDismiss={false}
+      dismissDirection="left"
+      onClose={function () {}}
+      style={position}
+      className="elevated"
+    >
+      This feature is in beta. If you have any suggestions or feeback, please{" "}
       <SubmitIssue
         href="https://docs.google.com/forms/d/e/1FAIpQLSde_zIFZPQD2p0ovaX3Pb7lDOajWJCmOeuX4wQ8Z8Ab5NXUjw/viewform"
         target="_blank"
         rel="noopener"
       >
-        quick survey
+        submit an issue on our GitHub
```

Comment:
We should either leave this with the survey copy or switch the link. 
cc: @hthomas-czi